### PR TITLE
Update on AST from SIR dependency removal in dawn

### PR DESCRIPTION
**Pull request by Stagno**
## Technical Description

Updating gtclang on partial AST -> SIR dependency removal.

## Dependencies

https://github.com/MeteoSwiss-APN/dawn/pull/251

### CODE REVIEW

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:Stagno/dawn.git"
```
Comment (mroethlin) [inline bundle/CMakeLists.txt]: Remember to change this back

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Pull request merged**